moveit_setup_assistant/templates/moveit_config_pkg_template/launch/ros_controllers.launch

```diff
@@ -6,12 +6,12 @@
 
   <!-- Load the controllers -->
   <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
-    output="screen" ns="/[GENERATED_PACKAGE_NAME]" args="spawn joint_state_controller position_trajectory_controller"/>
+    output="screen" ns="[ROBOT_NAME]" args="spawn joint_state_controller position_trajectory_controller"/>
```

Comment:
Looks like the ns prefix is very specific to your application. Why do you prefer `ROBOT_NAME` over `GENERATED_PACKAGE_NAME` now?

Reply:
After looking at [gazebo_ros_demos/rrbot_control.launch](https://github.com/ros-simulation/gazebo_ros_demos/blob/kinetic-devel/rrbot_control/launch/rrbot_control.launch) , I've found we use the robot name not the package name for the namespace.
Which is also done in the [ros_control tutorial here](http://gazebosim.org/tutorials/?tut=ros_control].